docs/application/native/guides/app-management/widget-service.md

```diff
@@ -0,0 +1,128 @@
+# Widget Service
+
+
+The Widget Service API provides functionality to get the information of installed widgets.
+
+The main features of the Widget Service API include:
+
+- Retrieving information of widgets
+
+  You can [retrieve the information](#retrieve) of a installed widget application.
+
+- Communicating with running widget instances
+
+  You can [communicate](#communicate) with running widget instances.
+
+
+
+> **Note**
+>
+> This API will only works if the widget and application are packaged in one.
+
+
+
+## Prerequisites
+
+1.  To use the Widget Service API (in [mobile](../../api/mobile/latest/group__CAPI__WIDGET__SERVICE__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__WIDGET__SERVICE__MODULE.html) applications), the application has to request permission by adding the following privilege to the `tizen-manifest.xml` file:
+
+    ```
+    <privileges>
+       <privilege>http://tizen.org/privilege/widget.viewer</privilege>
+    </privileges>
+    ```
+
+2.  To use the functions of the Widget Service API, include the `<widget_service.h>` header file in your application:
+
+    ```
+    #include <widget_service.h>
+    ```
+
+
+<a name="retrieve"></a>
+## Retrive the information
+
+Widget Service API provides functions to get various information of widget.
+
+1. Gets the pixel size of given size type.
+
+    ```
+    int ret;
+    int w;
+    int h;
+
+    ret = widget_service_get_size(WIDGET_SIZE_TYPE_2x2, &w, &h);
+    ```
+
+2. Gets the supported size list.
+
+    ```
```

Comment:
Global: 
Please specify the language identifier:

https://github.com/Samsung/tizen-docs/blob/master/styleguide/style.md#inline-code-blocks-with-language-identifier